### ISSUE TITLE
New setting "workbench.editor.labelFormat": "default" does not work (Linux)

### ISSUE THREAD

**Issue by fschoenm** [Microsoft/vscode#35721]
![screenshot_2017-10-06_15-19-17](https://user-images.githubusercontent.com/687230/31279654-261e3eb2-aaaa-11e7-8e2e-3ddf3cded4e9.png)

- VSCode Version: 1.17.0
- OS Version: Debian Linux

Steps to Reproduce:

1. Install latest VS Code version.
2. Tab labels with the same name don't show their parent folder anymore (see screenshot).

Reproduces without extensions: Yes

**Comment by fschoenm** [comment]
This worked in the previous version on Linux and also still seems to work on Windows.

**Comment by fschoenm** [comment]
I think the problem may be related to files that lie in the root directory of my open project. See the two screenshots below:

* In the first case, I opened two files with the same name but one of them is in the project root directory. 
![screenshot_2017-10-06_15-47-35](https://user-images.githubusercontent.com/687230/31280798-d4311bfc-aaad-11e7-95e8-679d503f61d0.png)

* In the second case, I opened another file with the same name. Now VS Code seems to recognize that there are multiple tabs with the same name and adds the parent directory.
![screenshot_2017-10-06_15-49-37](https://user-images.githubusercontent.com/687230/31280838-fff5afc8-aaad-11e7-9e90-6319522b7785.png)

This also happens on Windows.
